registrants/src/UI/embc-registrant/src/app/sharedModules/components/dashboard/dashboard.component.html

```diff
@@ -47,12 +47,28 @@
                 </div>
               }
               @if ((evacuationFilesDataService.hasPendingEssFiles$ | async) === false) {
```

Comment:
I think this if needs to go under to else if instead of nesting ifs - an evacuation file with supports will not be considered pending, it will be active